src/router/dispatcher.rs

```diff
@@ -88,7 +88,7 @@ pub struct Dispatcher {
     gateways: Vec<KeyedUri>,
     routing_height: u64,
     region_height: u64,
-    default_router: KeyedUri,
+    default_router: Option<KeyedUri>,
```

Comment:
OCD: I prefer seeing Option fields at the bottom of a struct

Reply:
OCD pacified!